src/main/java/pl/pjwstk/woloappapi/mapper/.gitkeep

```diff
@@ -0,0 +1 @@
+tu będą klasy dla mapowania obiektów entity <-> dto
```

Comment:
`.gitkeep` nie jest częścią gita, ale jest stosowanym wytrychem na to, aby zrobić odpowiednią strukturę katalogów (git nie pamięta katalogów, katalogi powstają wtórnie na bazie ścieżek do plików, które pamięta git. Jeśli folder jest pusty to nie ma pliku, który w swojej ścieżce ma ten folder)

https://www.freecodecamp.org/news/what-is-gitkeep/